util/src/main/java/tech/pegasys/artemis/util/config/Constants.java

```diff
@@ -191,7 +191,7 @@ public static void setConstants(String Constants) {
       CHURN_LIMIT_QUOTIENT = 65536;
       SHUFFLE_ROUND_COUNT = 10;
       MIN_GENESIS_ACTIVE_VALIDATOR_COUNT = 64;
-      MIN_GENESIS_TIME = UnsignedLong.valueOf(1578009600);
+      MIN_GENESIS_TIME = UnsignedLong.ONE;
```

Comment:
This is an interesting one.  The spec does say 1578009600 but that means the chain won't reach genesis until early 2020 which is really not useful for testing. So set it to a long time in the past - we probably need a way to override this to make testing easier but this will let us keep moving forward for now.

Reply:
What about creating a new set of constants for testing/development?

Reply:
We could but that's essentially what the minimal config is designed to be. I think it's just copied over the main net `MIN_GENESIS_TIME` value because there isn't really anything else sensible for it to have. Each network basically wants to set its own MIN_GENESIS_TIME anyway.

I suspect we may want to get to something more like what Besu does where there are some default network configs built in but you can also specify custom ones via a file (but possibly not the main config file or it gets too big and confusing).